gems/pending/appliance_console.rb

```diff
@@ -94,8 +96,24 @@
 require 'appliance_console/prompts'
 include ApplianceConsole::Prompts
 
+# Updates the ip address associated with a hostname in the /etc/hosts file
+#
+# @param host [String] The hostname to look for
+# @param ip [String] The address to write
+def update_hostname_ip(host, ip)
```

Comment:
I think this will go away entirely as we shouldn't be adding a line with the external IP when we could just resolve the local hostname to `127.0.0.1`.  This will also require a change to the process of setting a hostname described below.